### ISSUE TITLE
Transcriber self-certifies source of transcription was legally obtained

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/MyopicVicar#712]
Need to decide whether or not this is wanted (and then maybe implement)

**Comment by Captainkirkdawson** [comment]
Would it not be part of the Transcriber agreement?

**Comment by edickens** [comment]
We need to also allow for when we are donated transcriptions by
    non-transcribers.  In SOM when I was CC we were given permission to
    use the transcription freely available on the donor's personal
    website.  E
    

    On 13/01/2016 20:24, Kirk Dawson wrote:
    
    
      Would it not be part of the Transcriber agreement?
      —
        Reply to this email directly or view
          it on GitHub.

**Comment by PatReynolds** [comment]
Yes, Transcribers as part of TA.  What is the methodology for accepting others' transcriptions?

**Comment by edickens** [comment]
Methodology needs definition.
    
    At present, if offered a transcription or use of what is on their
    website, we ask if the transcription has been done by then, but do
    not ask too many questions as to whether they did it legally.  That
    is their problem and they sometime do not know the answer.  We
    credit them with the work so that we know the source.  We create
    (usually) a UserID to identify their work and keep it separate.  We
    have a number of people who are good at Excel and can usually
    convert the format without transcribing.  Job done.  E
    _____________________________________

    On 13/05/2016 17:56, PatReynolds wrote:
    
    
      Yes, Transcribers as part of TA. What is the methodology for
        accepting others' transcriptions?

**Comment by PatReynolds** [comment]
So, we have three questions that need asking, or not asking, at some point:
a) did you, individual or group who have transcribed outside of FreeREG, do so lawfully.  Eric has covered this in his reply.
b) did you, individual who has transcribed, within FreeREG, do so under an agreement with someone, which you or someone else negotiated, and which we hold on record and associate with your transcriptions?
c) did you, individual, transcribe in contravention of the terms and conditions under which you were given access to the materials?

The confusion between the three different questions, which has upset some volunteers, and overloaded the agreement with legal jargon means that it has been taken out of the agreement. 
(1) is now dealt with as Eric described. 
(2) I will shortly be writing to coords to ensure that this will be true for all transcriptions made under such an agreement, and we have a story to ensure that the agreement and the transcriptions are linked.
(3) can be dealt with on upload (i.e. this story) or as part of a code of conduct (which would be built bottom up rather than top down).  Which is preferred? Or something else?

**Comment by Sherlock21** [comment]
Just reading Pat's summary above, in isolation, raises in my mind the fact that the subject header of this item is all embracing. and not as specific to a particular type of Transcription. ( as in Post 3)
So to which condition is Pat's summary applying? 
and if only to the restricted set in post 3, then what's the situation for everything that is not within that subset?

**Comment by PatReynolds** [comment]
Summary is of 2 kinds of transcription (by others / by our volunteers).  Are there more kinds?

**Comment by edickens** [comment]
Everything probably falls under these headings.  But we do for
    example get permission to use the contents of another website, and
    we then skim it and reformat.  We get transcriptions donated and the
    CC converts and uploads.  We hope that we will get members of
    parishes to transcribe their own records and donate the result, but
    they should register as a transcriber.
    
    But the question is more to do with where they got the facts from
    that were used for the transcription.  Were they looking at
    FindMyPast?  Have they checked out the T&C of the source?  Have
    they acquired microfiche and if so were there T&C?
    
    E
    
      On 27/10/2016 14:49, PatReynolds wrote:
    
    
      Summary is of 2 kinds of transcription (by others / by our
        volunteers). Are there more kinds?
      —
        You are receiving this because you commented.
        Reply to this email directly, view
          it on GitHub, or mute
          the thread.
      
        
          
          
        
        
      
      {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/FreeUKGen/MyopicVicar","title":"FreeUKGen/MyopicVicar","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/FreeUKGen/MyopicVicar"}},"updates":{"snippets":[{"icon":"PERSON","message":"@PatReynolds in #712: Summary is of 2 kinds of transcription (by others / by our volunteers).  Are there more kinds?"}],"action":{"name":"View Issue","url":"https://github.com/FreeUKGen/MyopicVicar/issues/712#issuecomment-256645993"}}}

**Comment by edickens** [comment]
There is - if I am correct - another legal point here.  The 
transcription is the property of the transcriber.  As such they have 
rights over its use.  It is not up to us to know how they created it and 
from what.  That is their problem.  The more we dig into whether they 
have done something illegal only means that we are getting involved.  We 
just need to know that they have given us the right to use their work.

Of course with images that we obtain and ask our transcribers to work on 
is completely different.  We need to assure them that we have obtained 
the images legally and that we give them permission to transcribe them 
and they can use that transcription for whatever purpose they wish, or 
not if we need to impose restrictions.

E

**Comment by PatReynolds** [comment]
Modified title to reflect actual development required, i.e. 
either a tickybox on upload: 
Did you have the full permission of the owner or custodian of the image (or orginal, if transcribing from an original, or a photograph you took of an original) to transcribe for FreeREG.  If you only had permission to photogaph/transcribe for 'personal research' then please contact the Executive before uploading. Tick this box to certify you have all the permissions needed.

or a change to the code of conduct that 'I will not transcribe in contravention of the terms and conditions by which I have been given access to an image or the original, or permission to photograph from an original'

**Comment by richpomfret** [comment]
'By uploading you agree to....' statement should be included. Also add into code of conduct.

**Comment by PatReynolds** [comment]
Added to proposed revised code of conduct.  DRAFT: NEW: Legality: we ensure we do not break the law of our countries, or of England and Wales, in particular, we ensure that we have the permission to legally transcribe, and that we comply with prvacy laws, including ensuring that personal data is kept confidential through following  the Security Policy.

**Comment by Sherlock21** [comment]
regarding 'personal data' it will be necessary for you to define exactly which elements of a FreeReg record you refer to. 
Whose Security Policy do you refer to? if FR; then it might me useful to provide a navigable cross link to that?

**Comment by PatReynolds** [comment]
Link to the refinery page for clarity.
